### DISCUSSION TITLE
Blog index page automatically scrolls to top when click back after visiting a post/page

### DISCUSSION THREAD

**Discussion by kashifriazgh** [vercel/next.js#44530] [Help]
Hi, I have setup an application  using a startup created by NEXTJS version 13 and sanity.io v3. The blog application is amazingly fast but the problem is that when I (or possibly any other user) clicks and visit a post, and then goes back to main page (i.e. index page) , the index page automatically scroll to top instead of staying at same position where user left it.

The link of the demo application is here. please visit and check how this problem can be solved out.

https://sanity-tech-blog.vercel.app/

Please spend  some precious time to solve it out.

**Comment by ciruz** [comment]
Hey there was an issue with scroll restoration, which next.js version do you use?

There are other discussions and issues as well:
https://github.com/vercel/next.js/discussions/44013

There also was a fix in ``v13.0.7-canary.2``.
